src/metabase/xrays/automagic_dashboards/core.clj

```diff
@@ -876,7 +877,8 @@
                                                :dashboard-templates-prefix ["table"]})
                                             opts)]
                            (automagic-dashboard root'))
-                         (let [root'     (merge root
+                         (let [opts      (assoc opts :show max-cards-total)
```

Comment:
I'm not very familiar with the x-ray code, but I find it pretty hard to follow that we have these two levels where `show` can be set to a constant. Do you mind giving a brief outline of how the the flow works for "section" versus "total"?

Also, should we not code  bit more defensively here, and do something like `(merge {:show max-cards-total} opts)` or `(update opts :show (fnil min max-cards-total) max-cards-total)`?

Reply:
If you remember, that was the place where it said `(assoc opts :show :all)`, so I'm just bringing back some override :)